src/frontend/screens/ObservationEdit/ObservationEditView.js

```diff
@@ -137,7 +137,7 @@ const DescriptionField = () => {
           placeholderTextColor="silver"
           underlineColorAndroid="transparent"
           multiline
-          autoFocus={false}
+          autoFocus={true}
```

Comment:
My understanding of this requirement was that the description / notes field should _not_ autofocus (retaining current behaviour), but the individual questions that appear under "Add details" _should_ autofocus.